.github/actions/ginkgo/main-focus.yaml

```diff
@@ -214,6 +214,21 @@ include:
     cliFocus: "K8sKafkaPolicyTest"
 
 exclude:
+  # The bandwidth test is disabled and hubble tests are not meant
+  # to run on net-next.
+  - k8s-version: "1.29"
+    focus: "f10-agent-hubble-bandwidth"
+
+  # These tests are meant to run with kube-proxy which is not available
+  # with net-next
+  - k8s-version: "1.29"
+    focus: "f16-datapath-service-ew-2"
+
+  # These tests are meant to run with kube-proxy which is not available
+  # with net-next
+  - k8s-version: "1.29"
+    focus: "f17-datapath-service-ew-kube-proxy"
+
```

Comment:
can you share which criteria you decide which tests should be running in k8s 1.29?

Reply:
I just copied the ones from 1.28

Reply:
@sayboras thanks for the comment, I realized that I didn't change this file properly. It should be fixed now.